index.html

```diff
@@ -40,6 +40,129 @@ <h5>LET’S CONNECT</h5>
           <li><img src="imgs/Github.png" alt="Github Icon" /></li>
           <li><img src="imgs/hands.png" alt="hands" /></li>
         </ul><br><br><br><br><br><br><br>
+      </article>
+
+      <section class="grid-container">
+        <div class="grid-item show-case">
+          <div class="content">
+            <h4>Tonic</h4>
+            <ul class="flex">
+              <li class="cano">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
+        <div class="grid-item show-case2">
+          <div class="content">
+            <h4>Multi-Post Stories</h4>
+            <ul class="flex">
+              <li class="cano">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="flex ocontent">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit">See Project</button>
+        </div>
```

Comment:
- [ ] The button (e.g., "See project") is a link a or button tag, and follow the button styling: Active, Hover, and Pressed (see design). Kindly fix the Hover and pressed for your all buttons.
### Figma design
![figma-figma](https://user-images.githubusercontent.com/81727988/164911982-cd88afa6-cb2c-4b73-904b-4a6c796b7f7c.png)
### When user Hover your buttons
![your-hover](https://user-images.githubusercontent.com/81727988/164912002-4d3948a8-a828-4df9-84b0-d515046ef70d.png)
### when user pressed your buttons
![your-pressed](https://user-images.githubusercontent.com/81727988/164912023-d31a01df-da8f-4ede-b06c-dcabd72dc497.png)